docs/dg/dev/acp/acp-overview.md

```diff
@@ -9,13 +9,11 @@ redirect_from:
 keywords: acp
 ---
 
-The App Composition Platform (ACP) lets Spryker Cloud Commerce Operating System (SCCOS) customers connect, configure, and use the available third-party services via apps, in their application with the click of a button, with no or low development efforts from their side.
+The App Composition Platform (ACP) enables Spryker Cloud Commerce Operating System (SCCOS) customers connect, configure, and use the available third-party services available on the platform in their Spryker projects with zero or low development efforts required from their side.
 
-![ACP](https://spryker.s3.eu-central-1.amazonaws.com/docs/aop/app-orchestration-platform-overview/aop.png)
```

Comment:
Do we really want to remove this picture?

Reply:
Do you know why the picture was added? It's not clear the value it brings also AOP is outdated

Reply:
I see that SCCOS can obtain different payment/delivery methods based on what was chosen.
Nevertheless I'm OK to remove it, it looks too heavy and incomprehensible.